puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -23,29 +27,18 @@
     notify  => Exec["jenkins-jobs-update-${config_name}"],
   }
 
-  cron { "jenkins-jobs-update-${config_name}-delete-old":
-    ensure      => absent,
-    command     => "timeout 1h jenkins-jobs --conf ${inifile} update --delete-old ${directory}/${config_name} > /var/cache/jjb.xml",
-    hour        => 0,
-    minute      => 10,
-    environment => 'PATH=/bin:/usr/bin:/usr/sbin',
-    require     => File[$inifile],
+  git::repo { "jenkins-jobs-${config_name}":
```

Comment:
does this make the `file { "${directory}/${config_name}": …}` above obsolete? Or is that still used somewhere?